GitWorkflow/GitWorkflow/Level.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include "Platform.hpp"
+#include "EndPoint.hpp"
+#include "Player.hpp"
+
+class Level final : public sf::Drawable
+{
+public:
+	Level();
+	~Level() = default;
+
+	template<typename T, typename... V>
+	void create(const V& ...args)
+	{
+		if (std::is_same<T, Player>::value) m_player = std::make_unique<Player>(args...);
+		else if (std::is_same<T, EndPoint>::value) m_endPoint = std::make_unique<EndPoint>(args...);
+		else if(std::is_same<T, Platform>::value) m_platforms.push_back(std::make_unique<Platform>(args...));
```

Comment:
Use "if constexpr" to check in compile time.
Use std::is_same_v to retrieve value.